project/cmake/scripts/linux/install.cmake

```diff
@@ -14,35 +14,36 @@ else()
   set(USE_OPENGLES 0)
 endif()
 
-configure_file(${CORE_SOURCE_DIR}/project/cmake/${APP_NAME_LC}-config.cmake.in
-               ${CORE_BUILD_DIR}/${APP_NAME_LC}-config.cmake @ONLY)
-
 configure_file(${CORE_SOURCE_DIR}/tools/Linux/kodi.sh.in
                ${CORE_BUILD_DIR}/scripts/${APP_NAME_LC} @ONLY)
 configure_file(${CORE_SOURCE_DIR}/tools/Linux/kodi-standalone.sh.in
                ${CORE_BUILD_DIR}/scripts/${APP_NAME_LC}-standalone @ONLY)
 
-install(TARGETS ${APP_NAME_LC} DESTINATION ${libdir}/kodi)
+# cmake config
+set(APP_LIB_DIR ${CMAKE_INSTALL_PREFIX}/lib/${APP_NAME_LC})
+set(APP_PREFIX ${CMAKE_INSTALL_PREFIX})
+set(APP_INCLUDE_DIR ${CMAKE_INSTALL_PREFIX}/include/${APP_NAME_LC})
+set(CXX11_SWITCH "-std=c++11")
+configure_file(${PROJECT_SOURCE_DIR}/kodi-config.cmake.in
+               ${CORE_BUILD_DIR}/scripts/${APP_NAME_LC}-config.cmake @ONLY)
```

Comment:
The original version had the config generated to `BUILDDIR/kodi-config.cmake`. I decided to hide it in the scripts folder since it's only used for installation. When building a addon with the build dir, the user has to set `-DCMAKE_PREFIX_PATH` to `BUILDDIR/build`. It's just too easy to just to forget the `build` which would lead to the wrong config being picked up and weird errors.